src/containers/EditProfileContainer.js

```diff
@@ -16,20 +20,46 @@ const EditProfileContainer = props => {
     myUserInfo,
     uploadProfileImage,
   } = props
+
+  const [username, setUsername] = useState('')
+
+  useEffect(() => {
+    async function fetchUser() {
+      const user = await props.getUser(mmuserId)
+      setUsername(user.data.username)
+    }
+    fetchUser()
+  }, [])
+
   // Update profile picture
   const updatePicture = img => {
+    if (!img) {
+      props.setDefaultProfileImage(currentUser.id)
+    }
     if (currentUser && img) {
       uploadProfileImage(currentUser.id, dataUriToBlob(img))
     }
   }
 
-  const handleEditReady = (
+  const changeUsername = nickname => {
+    const letter = nickname[0].toLowerCase()
+    const firstRemoved = username.substr(1, 20)
+    const updated = letter.concat(firstRemoved)
+    return updated
+  }
+
+  const handleEditReady = async (
     newDescription,
     newNickname,
     showAge,
     showLocation,
-    location
+    location,
```

Comment:
We need to either update the picture here or in the EditProfile component.

Reply:
Moved the update to container.